### ISSUE TITLE
Blank Screen after publishing

### ISSUE THREAD

**Issue by painschen** [myflashlab/GoogleVR-ANE#8]
I'm using this small sample, but after publishing to my iphone7, i get only a white screen without error by trace()..

// import
import com.doitflash.consts._;
import com.doitflash.mobileProject.commonCpuSrc._;
import com.doitflash.starling.utils.list._;
import com.doitflash.text.modules._;
import flash.desktop._;
import flash.display._;
import flash.events._;
import flash.filesystem._;
import flash.text._;
import flash.ui._;
import com.myflashlab.air.extensions.googleVR.*;

// input
Multitouch.inputMode = MultitouchInputMode.GESTURE;

// stage
stage.scaleMode = StageScaleMode.NO_SCALE;
stage.align = StageAlign.TOP_LEFT;

// init
try{ 
    trace("VR INIT");
    VR.init(); 
    VR.api.addEventListener(VREvents.VR_CLOSED, onVRClosed);
    VR.api.addEventListener(VREvents.DID_TAP, onVRTapped);
    VR.api.addEventListener(VREvents.DISPLAY_MODE_CHANGED, onVRModeChanged);
}catch(e){ trace(e) };

// buttons
btnImg.addEventListener(MouseEvent.CLICK, function(e:MouseEvent){ startImg360(); })
btnVid.addEventListener(MouseEvent.CLICK, function(e:MouseEvent){ startVid360(); })

// img
function startImg360(){
    trace("launching a 360 image view..."); 
    var vrSettingImg:VRConfigImg = new VRConfigImg();
    vrSettingImg.image360 = File.applicationDirectory.resolvePath("360Img.jpg");
    vrSettingImg.btnExit = File.applicationDirectory.resolvePath("exit.png");  
        vrSettingImg.imageType = VRConfig.TYPE_STEREO_OVER_UNDER;
        vrSettingImg.stereoModeButtonEnabled = false;
        vrSettingImg.infoButtonEnabled = true;
        vrSettingImg.touchTrackingEnabled = false;  
    VR.api.imageView.addEventListener(VREvents.LOAD_FAILED, onLoadFailed);
    VR.api.imageView.addEventListener(VREvents.LOAD_SUCCESS, onLoadSuccess);
    VR.api.imageView.attach(vrSettingImg);
}
function onLoadFailed(e:VREvents):void{
    VR.api.imageView.removeEventListener(VREvents.LOAD_FAILED, onLoadFailed);
    VR.api.imageView.removeEventListener(VREvents.LOAD_SUCCESS, onLoadSuccess); 
    trace("onLoadFailed: " + e.msg);
}
function onLoadSuccess(e:VREvents):void{
    VR.api.imageView.removeEventListener(VREvents.LOAD_FAILED, onLoadFailed);
    VR.api.imageView.removeEventListener(VREvents.LOAD_SUCCESS, onLoadSuccess); 
    trace("onLoadSuccess");
}

// video  
function startVid360(){
    VR.api.videoView.removeEventListener(VREvents.LOAD_FAILED,  onVideoLoadFailed);
    VR.api.videoView.removeEventListener(VREvents.LOAD_SUCCESS, onVideoLoadSuccess);  
    VR.api.videoView.removeEventListener(VREvents.PAUSED,       onVideoPaused);
    VR.api.videoView.removeEventListener(VREvents.REACHED_END,  onVideoReachedEnd);
    VR.api.videoView.removeEventListener(VREvents.RESUMED,      onVideoResumed);
    VR.api.videoView.removeEventListener(VREvents.STOPPED,      onVideoStopped);
    VR.api.videoView.removeEventListener(VREvents.POS_UPDATED,  onVideoPositionUpdated);  
    trace("launching a 360 Video view...");
    var vrSettingVid:VRConfigVid = new VRConfigVid();
    vrSettingVid.video360 = File.applicationDirectory.resolvePath("360Vid.mp4");
    vrSettingVid.btnExit = File.applicationDirectory.resolvePath("exit.png");
        vrSettingVid.videoType = VRConfig.TYPE_STEREO_OVER_UNDER;
        vrSettingVid.stereoModeButtonEnabled = true;
        vrSettingVid.infoButtonEnabled = true;
        vrSettingVid.touchTrackingEnabled = false;
        vrSettingVid.fullscreenButtonEnabled = true;  
    VR.api.videoView.addEventListener(VREvents.LOAD_FAILED,     onVideoLoadFailed);
    VR.api.videoView.addEventListener(VREvents.LOAD_SUCCESS,    onVideoLoadSuccess);  
    VR.api.videoView.addEventListener(VREvents.PAUSED,          onVideoPaused);
    VR.api.videoView.addEventListener(VREvents.REACHED_END,     onVideoReachedEnd);
    VR.api.videoView.addEventListener(VREvents.RESUMED,         onVideoResumed);
    VR.api.videoView.addEventListener(VREvents.STOPPED,         onVideoStopped);
    VR.api.videoView.addEventListener(VREvents.POS_UPDATED,     onVideoPositionUpdated);  
    VR.api.videoView.attach(vrSettingVid);
}
function onVideoLoadFailed(e:VREvents):void{
    VR.api.videoView.removeEventListener(VREvents.LOAD_FAILED, onVideoLoadFailed);
    VR.api.videoView.removeEventListener(VREvents.LOAD_SUCCESS, onVideoLoadSuccess);  
    trace("onVideoLoadFailed: " + e.msg);
}
function onVideoLoadSuccess(e:VREvents):void{
    VR.api.videoView.removeEventListener(VREvents.LOAD_FAILED, onVideoLoadFailed);
    VR.api.videoView.removeEventListener(VREvents.LOAD_SUCCESS, onVideoLoadSuccess);  
    trace("onVideoLoadSuccess, video duration = " + e.duration);
}
function onVideoPaused(e:VREvents):void{
    trace("onVideoPaused");
}
function onVideoResumed(e:VREvents):void{
    trace("onVideoResumed");
}
function onVideoStopped(e:VREvents):void{
    trace("onVideoStopped");
}
function onVideoReachedEnd(e:VREvents):void{
    VR.api.videoView.seekTo(0);
    VR.api.videoView.resume();  
    trace("onVideoReachedEnd");
}
function onVideoPositionUpdated(e:VREvents):void{
    trace("position updated... " + e.position + " / " + e.duration);
}

// vr
function onVRClosed(e:VREvents):void{
    trace("vr window closed!");
}
function onVRTapped(e:VREvents):void{
    trace("vr window tapped!"); 
    trace(VR.api.headRotation); 
    if (VR.api.videoView.isPaused){
        VR.api.videoView.resume();
    } else {
        VR.api.videoView.pause();
    }
}
function onVRModeChanged(e:VREvents):void{
    switch (e.displaymode) {
        case DisplayMode.EMBEDDED:
            trace("vr mode changed: EMBEDDED");
        break;  
        case DisplayMode.FULLSCREEN_MONO:
            trace("vr mode changed: FULLSCREEN_MONO");
        break;  
        case DisplayMode.FULLSCREEN_STEREO:
            trace("vr mode changed: FULLSCREEN_STEREO");
        break;  
        default:
    }
}

![bildschirmfoto 2016-10-14 um 10 49 44](https://cloud.githubusercontent.com/assets/4842379/19381495/17d2eab4-91fc-11e6-87d0-5fa7a544388a.png)
![bildschirmfoto 2016-10-14 um 10 50 15](https://cloud.githubusercontent.com/assets/4842379/19381494/17d251ee-91fc-11e6-9573-eb414e17df39.png)
![bildschirmfoto 2016-10-14 um 10 51 37](https://cloud.githubusercontent.com/assets/4842379/19381524/47f8f198-91fc-11e6-96fc-230decb5fe0f.png)

**Comment by painschen** [comment]
i noticed, that the image works only, if i started the video before.. so if i start video, click on the exit arrow and start image, it works.. why? :o

**Comment by painschen** [comment]
hm it works if i disable tracking..

**Comment by painschen** [comment]
so now i notice, that sometimes the video is showing a blank screen, too..

**Comment by myflashlab** [comment]
I think we would need more information to be able to help you more exactly. If you can send us xcode logs while your device is connected to your mac it would be very helpful.

I wonder if you are using GPU in your AIR app before starting the VR experience? for example if you are running a Starling content, you need to stop it prior to running the VR ANE.

**Comment by dgurney75** [comment]
I've tested this as well on an iPhone 7 running ios 10.2.1 and can verify that when using the demo ANE it will cover the entire video or image with a white screen. When you tap the home button and close the app and then try to re-open it, you will be able to view the video or image for about 1 second before it's covered again. Hope that helps. Using AIR 24.0.0.180 and Adobe Animate CC to test with.

**Comment by dgurney75** [comment]
Any update on this?

**Comment by zanorg** [comment]
Hi, I have the same problem on Android.
Using Adobe Animate CC, when I test the app on my android phone I just see a blank screen, with console on top (testing the Demo).
If I click on Image 360, log is "launching a 360 image view..."  but nothing happens.
Same thing for video, nothing special if I start with video or image.
In debug nothing appears, no "Load Fail" or "Load Success".

I haven't done anything special, just testing the free ANE in order to purchase it if it works fine (a client just need to view an image and/or video). By the way I did not have any button saying "it's a free version, click ok to continue".
Tested with AIR 24 & 25.
